dpe/src/commands/derive_child.rs

```diff
@@ -7,53 +7,63 @@ use crate::{
     tci::TciMeasurement,
     DPE_PROFILE,
 };
+use bitflags::bitflags;
+
+#[repr(C)]
+#[derive(Debug, PartialEq, Eq, zerocopy::FromBytes)]
+#[cfg_attr(test, derive(zerocopy::AsBytes))]
+pub struct DeriveChildFlags(u32);
+
+bitflags! {
+    impl DeriveChildFlags: u32 {
+        const INTERNAL_INPUT_INFO = 0b10000000_00000000_00000000_00000000;
```

Comment:
(here and elsewhere) I think it would be more clear to define these in the style `1u32 << 31`